app/routes/events/EventCreateRoute.ts

```diff
@@ -68,6 +70,8 @@ const mapStateToProps = (state, props) => {
     },
     pools: valueSelector(state, 'pools'),
   };
+  const currentUser: DetailedUser = selectCurrentUser(state);
```

Comment:
`EventCreateRoute` (and therefore also `EventEditor`) has already access to `currentUser` through its props. See app/routes/events/index.tsx

Also, the correct type is `CurrentUser`, not `DetailedUser`, but that doesn't matter since you can just remove this.